backend/data/src/main/kotlin/io/tolgee/email/EmailGlobalVariablesProvider.kt

```diff
@@ -27,14 +28,16 @@ class EmailGlobalVariablesProvider(
   // Used to identify whether we're Tolgee Cloud or not
   private val billingConfigProvider: PublicBillingConfProvider,
   private val tolgeeProperties: TolgeeProperties,
+  private val frontendUrlProvider: FrontendUrlProvider,
 ) {
   operator fun invoke(): Map<String, Any?> {
     val isCloud = billingConfigProvider().enabled
+    val backendUrl = tolgeeProperties.backEndUrl ?: frontendUrlProvider.url
 
     return mapOf(
       "isCloud" to isCloud,
-      "instanceQualifier" to if (isCloud) tolgeeProperties.appName else tolgeeProperties.frontEndUrl.intoQualifier(),
-      "instanceUrl" to tolgeeProperties.frontEndUrl,
+      "instanceQualifier" to if (isCloud) tolgeeProperties.appName else backendUrl.intoQualifier(),
+      "backendUrl" to backendUrl
```

Comment:
Unsure about that... Especially wrt the instance qualifier. The fallback to using the domain name people are using to access Tolgee is likely a lot more recognizable than whatever internal URL is used.

Problems in development environment are likely related to the fact I completely overlooked that and didn't add the appropriate proxy config to Vite 😅